llvm/docs/CommandGuide/llvm-debuginfo-analyzer.rst

```diff
@@ -1952,6 +1953,316 @@ The **{Coverage}** and **{Location}** attributes describe the debug
 location and coverage for logical symbols. For optimized code, the
 coverage value decreases and it affects the program debuggability.
 
+WEBASSEMBLY SUPPORT
+~~~~~~~~~~~~~~~~~~~
+The below example is used to show the WebAssembly output generated by
+:program:`llvm-debuginfo-analyzer`. We compiled the example for a
+WebAssembly 64-bit target with Clang (-O0 -g --target=wasm64):
```

Comment:
wasm64 is not standardized yet so it might be best to stick to wasm32 for the example for now.

Reply:
Changed to use wasm32.